src/index.html

```diff
@@ -1,12 +1,304 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="/styles/main.scss">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="container">
+      <div class="container_navigation-logo">
+        <a href="#home">
+          <img 
+            src="./images/logo.png" 
+            alt="#home"
+            href="#back-home"
+          >
+        </a>
+      </div>
+
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a href="#" class="nav_link is-active">Apple</a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link">Samsung</a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link">Смартфоны и телефоны</a>
+          </li>
+          <li class="nav_item">
+            <a 
+              href="#"
+              class="nav_link"
+              data-qa="nav-hover"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link">Гаджеты</a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link">Планшеты</a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link">Фото</a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <div class="container-card">
+        <div class="container-card_product-card product-card">
+          <img
+            class="image product-card_image"
+            src="./images/imac.jpeg" 
+            alt="image"
+          >
+          <h3 class="product-card_description">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h3>
+          <p class="product-card_item-code">Код товара: 195434</p>
+          <div class="product-card_container-mark container-mark" data-qa="card_hover">
+            <div class="container-mark_stars container-mark_stars--4"></div>
+            <div class="container-mark_stars container-mark_stars--4"></div>
+            <div class="container-mark_stars container-mark_stars--4"></div>
+            <div class="container-mark_stars container-mark_stars--4"></div>
+            <div class="container-mark_stars"></div>
+            <p class="container-mark_reviews">Отзывов: 5</p>
+          </div>
```

Comment:
```suggestion
          <div class="product-card__container-mark container-mark container-mark--4" data-qa="card_hover">
            <div class="container-mark__star"></div>
            <div class="container-mark__star"></div>
            <div class="container-mark__star"></div>
            <div class="container-mark__star"></div>
            <div class="container-mark__star"></div>
            <p class="container-mark_reviews">Отзывов: 5</p>
          </div>
```
Класс модификатор должен быть только у контейнера звездочек, а не у них самих. Используй код из предыдущего задания